core/src/test/scala/unit/kafka/server/MetadataRequestTest.scala

```diff
@@ -35,6 +35,26 @@ class MetadataRequestTest extends BaseRequestTest {
   }
 
   @Test
+  def testClusterIdWithRequestVersion2() {
```

Comment:
Does this test add any value over `testClusterIdIsValid`? It doesn't seem to me that it does.